Rakefile

```diff
@@ -5,7 +5,9 @@ require "simplecov"
 require_relative "config/application"
 
 Rails.application.load_tasks
+Rake::Task["default"].clear 
 
 task lint: ["lint:ruby"]
 task annotate: ["db:annotate"]
-task default: %i[spec annotate lint brakeman]
+task parallel: ["parallel:spec"]
+task default: %i[parallel annotate lint brakeman]
```

Comment:
Do we want to have a separate explicit PR for moving these tests to now be parallel? :eyes: or at least a commit, feels like a significant change

Reply:
Yep good idea - will make it a separate commit